qcodes/instrument_drivers/tektronix/Keithley_2450.py

```diff
@@ -229,7 +244,7 @@ def __init__(self, parent: 'Keithley2450', name: str, proper_function: str) -> N
             get_cmd=self._measure_sweep,
             unit=unit,
             vals=Arrays(shape=(self.parent.npts,)),
-            parameter_class=NewParameterWithSetpoints
+            parameter_class=ParameterWithSetpointsAllType
```

Comment:
sorry, i forgot - what are the setpoints of this parameter-with-setpoints? is it up to the user to decide/define?

Reply:
The "setpoints" will be defined by the user, with start, end, step count, etc... when defining the sense and source function, there are:
> sense.sweep.setpoints = (self.source.sweep_axis,)

and

>self.sense.sweep.setpoints = (source.sweep_axis,)

where the sweep_aixs is:

>     def get_sweep_axis(self) -> np.ndarray:
>         if self._sweep_arguments == {}:
>             raise ValueError(
>                 "Please setup the sweep before getting values of this parameter"
>             )
> 
>         return np.linspace(
>             start=self._sweep_arguments["start"],
>             stop=self._sweep_arguments["stop"],
>             num=self._sweep_arguments["step_count"]
>         )